winml/lib/Api.Image/EventTimer.h

```diff
@@ -0,0 +1,29 @@
+#include "pch.h"
+
+class EventTimer
+{
+public:
+  EventTimer()
```

Comment:
May it be EventTimer() == default or not even declare it at all?
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rc-zero

Reply:
good point.